CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+
+
+Highlights:
+
+In Skaffold v2 what was previously the `deploy` phase of Skaffold is now split into a new `render` phase and `deploy` phase.  This clear boundary of separation between render and deploy phases allowed the team to simplify our code and CLI allowing us to clean up previously confusing or redundant flags like `skaffold deploy --render-only`, `skaffold deploy --skip-render`. 
+This release comes with a new schema version `v3alpha1`. This schema introduced a new `manifests` section which declares all resources an application deploys e.g helm charts, kubernetes yaml, kustomize directories and kpt configuration. This decoupling of manifests declaration from the deploy section allows manifests to be used across deploy tools e.g.
+* you can configure kpt deployer to render and apply kubernetes yaml, helm charts or
+* you can configure the kubectl deployer to apply helm charts and helm to render the charts. 
+
+New Features and Additions:
+* More advanced and configurable `render` functionality.  See the render docs [here](https://skaffold-v2.web.app/docs/pipeline-stages/renderers/) for more details.
+    * Skaffold v2 supports rendering, validating and transforming manifests with kpt v1.0.0-beta. See the kpt renderer docs [here](https://skaffold-v2.web.app/docs/pipeline-stages/renderers/kpt/) for more details.
+    * Please take a look at the kpt v0.39.0 to kptv1.0.0 migration guide of more information
+* Better rendering support for applications with helm charts. See the helm renderer docs [here](https://skaffold-v2.web.app/docs/pipeline-stages/renderers/helm/) for more details.
+    * skaffold under the hood uses `helm template` to render helm charts now
+* New `skaffold verify` command which allows for Skaffold to run test containers against skaffold deployments.  See the verify docs [here](https://skaffold-v2.web.app/docs/verify) for full set of details and examples. With this users can use off the shelf (`alpine`, etc.) or skaffold built test containers and run them in a pipeline that skaffold watches and streams logs from. Supports customer requested CI/CD functionality for a Skaffold native way of supporting:
+health checks (deployment success, readiness)
+integration/smoke/load tests
+monitoring checks(metrics and alerting)
+* beta `Cloud Run` deployer support.  Currently supports deploying applications, port-forwarding and eventing but not log streaming which will be added soon. See the docs [here](https://skaffold-v2.web.app/docs/pipeline-stages/deployers/cloudrun/) for more information
+
+Docs, Test, and Release Updates:
+* For the skaffold v2 beta period, there will be a temporary v2 docs site hosted at [https://skaffold-v2.web.app/](https://skaffold-v2.web.app/) which explains all changes, new features, and potential regressions.
```

Comment:
should we add a change to the skaffold.dev landing page which points to this url?
e.g. For skaffold V2 docs, visit https://skaffold-v2.web.app

Reply:
Yes, great idea.  I can do that in a followup PR